source/components/Dropdown/index.js

```diff
@@ -16,14 +16,20 @@ class Dropdown extends React.Component {
 
         this.state = {
             isClicked: false,
-            isTouchDevice: typeof window !== 'undefined' && ('ontouchstart' in window),
+            isTouchDevice: false,
         };
 
         this.onClick = this.onClick.bind(this);
         this.onToggleClicked = this.onToggleClicked.bind(this);
         this.onMouseLeave = this.onMouseLeave.bind(this);
     }
 
+    componentDidMount() {
+        this.setState({
+            isTouchDevice: typeof window !== 'undefined' && ('ontouchstart' in window),
```

Comment:
the only odd thing about having it here
is that any time you'll render Dropdown component you'll check if it is still a touch device...
@xkxd @latata is there a solution to this? or is it just how it is?

Reply:
We'll go with this solution to unblock development since we're wrapping up dropdowns, but let's keep discussion open about patterns like this.

This code is moved to componentDidMount as we know for sure it runs immediately component is mount on the browser side - this way we overwrite the isTouchDevice value which is set to false on the SSR side immediately after rendering the component.